panda_rover/detection_bound[] detections